docker-compose.yml

```diff
@@ -6,15 +6,6 @@ services:
     # image: freqtradeorg/freqtrade:develop
     # Use plotting image
     # image: freqtradeorg/freqtrade:develop_plot
-    # # Enable GPU Image and GPU Resources (only relevant for freqAI)
```

Comment:
This seems like a merge gone bad/wrong (accepting the wrong version).

It's reverting a change we did recently - and i assume it's the same case with several other files, too (this wasn't that big of a PR).

Reply:
reverted to your state, should be fixed now. sorry